Analyzer/interface/TupleMaker.h

```diff
@@ -206,17 +206,17 @@ void TupleMaker::initializeTuple(Tuple *&tuple,
                                  bool isSignal,
                                  unsigned int NCuts,
                                  unsigned int NCuts_Flip,
-                                 double PtHistoUpperBound,
-                                 double MassHistoUpperBound,
+                                 float PtHistoUpperBound,
```

Comment:
@tvami 
By construction, min and max bins of TH1  are double. I am wondering if we could not keep those lower/upper edges as double-s?

Reply:
but we really dont need double-s, these numbers are not even revant upto one digit after the 0